src/main/java/com/aerospike/load/AerospikeLoad.java

```diff
@@ -312,6 +314,10 @@ private static void initReadWriteThreadCnt(CommandLine cl) {
 		nReaderThreads = (nReaderThreads > 0
 				? (nReaderThreads > Constants.MAX_THREADS ? Constants.MAX_THREADS : nReaderThreads) : 1);
 		log.debug("Using reader Threads: " + nReaderThreads);
+
+		// add 1 for the tend thread
+		maxConnsPerNode = nWriterThreads + nReaderThreads + 1;
```

Comment:
I am confused by the log line. Won't this add a thread for each node? So if we have 3 nodes this will add 3 extra threads even though you only need a single thread. Maybe it is not important that it is exact

Reply:
The worker thread count does not change based on node count. The log line reads "Max connections per node: " just because that is the client config it maps to.